test/cli/iGenVar_cli_test.cpp

```diff
@@ -224,6 +268,33 @@ TEST_F(detect_breakends, with_detection_method_duplicate_arguments)
         "Methods to be used: [cigar_string,cigar_string]\n"
     };
     EXPECT_NE(result.exit_code, 0);
-    EXPECT_EQ(result.out, "");
+    EXPECT_EQ(result.out, std::string{});
+    EXPECT_EQ(result.err, expected_err);
+}
+
+TEST_F(detect_breakends, test_direct_methods_input)
+{
+    cli_test_result result = execute_app("iGenVar",
+                                         "-j", data(default_alignment_long_reads_file_path),
+                                         fasta_out_file_path,
+                                         "-m 0 -m 1 -c 0 -r 0");
+    EXPECT_EQ(result.exit_code, 0);
+    EXPECT_EQ(result.out, expected_res);
+    EXPECT_EQ(result.err, expected_err_default_no_err);
+}
+
+//TODO (irallia): This should get a better Error message with resolving https://github.com/seqan/seqan3/issues/2464
+TEST_F(detect_breakends, test_unknown_argument)
+{
+    cli_test_result result = execute_app("iGenVar",
+                                         "-j", data(default_alignment_long_reads_file_path),
+                                         fasta_out_file_path,
+                                         "-m 9");
+    std::string expected_err
+    {
+        "[Error] Value parse failed for -m: Argument 9 could not be parsed as type std::string.\n"
+    };
```

Comment:
This unhelpful error message is fixed by issue https://github.com/seqan/seqan3/issues/2464.